new-arch-migration/README.md

```diff
@@ -54,14 +54,16 @@ You need to be an administrator of the GitGuardian namespace where the applicati
 
     If needed, specify the Kubernetes namespace with `--namespace` (default namespace is used if not specified).
 
-3. Clone the ggtool repository.
+3. Configure RBAC permissions on the cluster as per the instructions provided in the [Kubernetes Application RBAC documentation](https://docs.gitguardian.com/self-hosting/installation/installation-existing-cluster#kubernetes-application-rbac) page.
+
+4. Clone the ggtool repository.
```

Comment:
`ggtool` should be named `ggtools` everywhere to avoid confusion, since the repository name is `ggtools`